compose/material3/material3/src/androidMain/kotlin/androidx/compose/material3/ExposedDropdownMenu.kt

```diff
@@ -205,82 +205,45 @@ private class OnGlobalLayoutListener(
     }
 }
 
-/**
- * Scope for [ExposedDropdownMenuBox].
- */
-@ExperimentalMaterial3Api
-interface ExposedDropdownMenuBoxScope {
-    /**
-     * Modifier which should be applied to a [TextField] (or [OutlinedTextField]) placed inside the
-     * scope. It's responsible for properly anchoring the [ExposedDropdownMenu], handling semantics
-     * of the component, and requesting focus.
-     */
-    fun Modifier.menuAnchor(): Modifier
-
-    /**
-     * Modifier which should be applied to an [ExposedDropdownMenu] placed inside the scope. It's
-     * responsible for setting the width of the [ExposedDropdownMenu], which will match the width of
-     * the [TextField] (if [matchTextFieldWidth] is set to true). It will also change the height of
-     * [ExposedDropdownMenu], so it'll take the largest possible height to not overlap the
-     * [TextField] and the software keyboard.
-     *
-     * @param matchTextFieldWidth whether the menu should match the width of the text field to which
-     * it's attached. If set to `true`, the width will match the width of the text field.
-     */
-    fun Modifier.exposedDropdownSize(
-        matchTextFieldWidth: Boolean = true
-    ): Modifier
-
-    /**
-     * Popup which contains content for Exposed Dropdown Menu. Should be used inside the content of
-     * [ExposedDropdownMenuBox].
-     *
-     * @param expanded whether the menu is expanded
-     * @param onDismissRequest called when the user requests to dismiss the menu, such as by
-     * tapping outside the menu's bounds
-     * @param modifier the [Modifier] to be applied to this menu
-     * @param content the content of the menu
-     */
-    @Composable
-    fun ExposedDropdownMenu(
-        expanded: Boolean,
-        onDismissRequest: () -> Unit,
-        modifier: Modifier = Modifier,
-        content: @Composable ColumnScope.() -> Unit
-    ) {
-        // TODO(b/202810604): use DropdownMenu when PopupProperties constructor is stable
-        // return DropdownMenu(
-        //     expanded = expanded,
-        //     onDismissRequest = onDismissRequest,
-        //     modifier = modifier.exposedDropdownSize(),
-        //     properties = ExposedDropdownMenuDefaults.PopupProperties,
-        //     content = content
-        // )
-
-        val expandedStates = remember { MutableTransitionState(false) }
-        expandedStates.targetState = expanded
-
-        if (expandedStates.currentState || expandedStates.targetState) {
-            val transformOriginState = remember { mutableStateOf(TransformOrigin.Center) }
-            val density = LocalDensity.current
-            val popupPositionProvider = DropdownMenuPositionProvider(
-                DpOffset.Zero,
-                density
-            ) { anchorBounds, menuBounds ->
-                transformOriginState.value = calculateTransformOrigin(anchorBounds, menuBounds)
-            }
+@Composable
+actual fun ExposedDropdownMenuBoxScope.ExposedDropdownMenu(
```

Comment:
This will break binary backwards-compatibility on Android; not sure how important it is, though. @igordmn can say better.

Reply:
Extracted expect/actual into internal function